ProcessLib/Parameter/CurveScaledParameter.h

```diff
@@ -35,32 +35,31 @@ struct CurveScaledParameter final : public Parameter<T> {
     {
         _parameter =
             &findParameter<T>(_referenced_parameter_name, parameters, 0);
-        _cache.resize(_parameter->getNumberOfComponents());
     }
 
     int getNumberOfComponents() const override
     {
         return _parameter->getNumberOfComponents();
     }
 
-    std::vector<T> const& operator()(double const t,
+    std::vector<T> operator()(double const t,
                                      SpatialPosition const& pos) const override
     {
         auto const& tup = (*_parameter)(t, pos);
         auto const scaling = _curve.getValue(t);
 
         auto const num_comp = _parameter->getNumberOfComponents();
+        std::vector<T> cache(num_comp);
```

Comment:
May be this change makes a little slow down in the performance. I checked the maximum Total Test time (real) in the building history:
This PR: 494 s
PR #2334:  464.40 s
PR #2338:  457.47 s

Reply:
The numbers are not very reliable because of changing load while tests run.